src/test/java/teammates/test/cases/datatransfer/AccountAttributesTest.java

```diff
@@ -238,15 +239,14 @@ private AccountAttributes createValidAccountAttributesObject() {
     }
 
     private AccountAttributes createAccountAttributesToSanitize() {
-
-        return AccountAttributes.builder()
-                .withGoogleId("googleId@gmail.com")
-                .withName("'name'")
-                .withInstitute("\\/")
-                .withEmail("&<email>&")
-                .withIsInstructor(true)
-                .build();
-
+        AccountAttributes unsanitizedAttributes = AccountAttributes.builder().build();
+        unsanitizedAttributes.googleId = "    google'Id@gmail.com\t";
+        unsanitizedAttributes.name = "'name'\n\n";
```

Comment:
There should be some extra space inside name in order to make sure `sanitizeName` is doing the job? `    'name'\n\n`?